workflow/scripts/add_extra_components.py

```diff
@@ -882,6 +883,346 @@ def trim_network(n, trim_topology):
     n.determine_network_topology()
 
 
+def add_co2_storage(n: pypsa.Network, config: dict, co2_storage_csv: str, costs: pd.DataFrame, sector: bool):
+    """Adds node level CO2 (underground) storage."""
+    # get node level CO2 (underground) storage potential and cost from CSV file
+    co2_storage = pd.read_csv(co2_storage_csv).set_index("node")
+
+    # add buses to represent node level CO2 captured by different processes
+    n.madd(
+        "Bus",
+        co2_storage.index,
+        suffix=" co2 capture",
+        carrier="co2",
+    )
+
+    # add stores to represent node level CO2 (underground) storage
+    n.madd(
+        "Store",
+        co2_storage.index,
+        suffix=" co2 storage",
+        bus=co2_storage.index + " co2 capture",
+        e_nom_extendable=True,
+        e_nom_max=co2_storage["potential [MtCO2]"] * 1e6,  # in tCO2
+        marginal_cost=co2_storage["cost [USD/tCO2]"],
+        carrier="co2",
+    )
+
+    # add carrier to represent CO2
+    n.madd(
+        "Carrier",
+        ["co2"],
+        color=config["plotting"]["tech_colors"]["co2"],
+        nice_name=config["plotting"]["nice_names"]["co2"],
+    )
+
+    # add carrier to represent CC only (i.e. without S)
+    carriers = n.carriers.query("Carrier.str.endswith('CCS')")
+    if carriers.empty is False:
```

Comment:
Kinda picky, but probably better to use `if not carriers.empty:` following the [Programming Recommendations in PEP8](https://peps.python.org/pep-0008/#programming-recommendations)? 

```suggestion
    if carriers.empty is False:
    if not carriers.empty:
```

Reply:
This review/feedback is solved/implemented with commit https://github.com/ricnogfer/pypsa-usa/commit/3854dfaf85bad703b949998ae4b75aa43c64cccd.